controller/controller.go

```diff
@@ -121,6 +121,8 @@ func respondWithError(w http.ResponseWriter, err error) {
 }
 
 func appHandler(c handlerConfig) http.Handler {
+	loadSchemas()
```

Comment:
Well I'd keep it in the same location, just change the package declaration at the top of the file to `schema`